subprojects/language-groovy/src/main/java/org/gradle/api/tasks/compile/GroovyCompile.java

```diff
@@ -103,17 +112,15 @@ private boolean experimentalCompilationAvoidanceEnabled() {
     @TaskAction
     protected void compile() {
         checkGroovyClasspathIsNonEmpty();
-        separateCompileClasspathIfCompileAvoidanceEnabled();
+        warnIfCompileAvoidanceEnabled();
         DefaultGroovyJavaJointCompileSpec spec = createSpec();
         WorkResult result = getCompiler(spec).execute(spec);
         setDidWork(result.getDidWork());
     }
 
-    private void separateCompileClasspathIfCompileAvoidanceEnabled() {
+    private void warnIfCompileAvoidanceEnabled() {
         if (experimentalCompilationAvoidanceEnabled()) {
             DeprecationLogger.incubatingFeatureUsed("Groovy compilation avoidance is an incubating feature.");
```

Comment:
Shouldn't this be `Groovy compilation avoidance`? The `is an incubating feature.` should be added by the method, right?

Reply:
Nice catch. I wasn't aware of that until checking out the code. Thanks. fixed.